analyzer/src/main/kotlin/managers/utils/MavenSupport.kt

```diff
@@ -177,6 +178,15 @@ class MavenSupport(workspaceReader: WorkspaceReader) {
                             VcsInfo(type = VcsType.SUBVERSION, url = url, revision = "", path = path)
                         }
 
+                        type == "http" || type == "https" -> {
```

Comment:
What about checking instead if the url starts with "//"? This would also catch entries like this one I saw recently:
`<connection>scm:git://github.com/vigna/fastutil.git</connection>`

Reply:
Good idea, will do that.